src/Chapter1.hs

```diff
@@ -620,8 +625,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let last2Digits = mod (abs n) 100
+      tens = quot last2Digits 10
+   in tens + (lastDigit last2Digits)
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that is actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (mod x 100) 10
```

You can see how we could pattern match on the pair 🙂 

Similar, there's `quotRem` as an analogue to `divMod`. The `div/mod` pair has different behaviour from `quotRem`, especially around negative numbers. So this corner case is something to keep in mind :slightly_smiling_face: